using-valtimo/document/assigning-a-user.md

```diff
@@ -0,0 +1,47 @@
+# Assigning a user to a document
+
+Every document can now be assigned to a user.
+
+## Prerequisites
+The user must have the correct role to access case types. 
+This is the case when the case type is visible in the `Cases` menu
+
+## Assigning using the user interface
+In the menu `Cases` a case type must be selected and then a specific case must be opened.
+On this page a `Claim` button is visible on the right side of the page. When the current 
+user is not yet assigned to a document this button will be enabled. Pressing this button 
+will assign the document to the user pressing the button
+
+![Assigning user with claim button](img/assigning-user-claim-button.png)
+
+## Assigning using a proces
+It is possible to assign a user to the current process by creating a service task and add the following expression
+
+```
+${documentDelegate.setAssignee(execution, "email address of user")}
+```
+
+When this service task is reached the assignee will be set
+
+![Assign user in process](img/assign-user-in-process.png)
+
```

Comment:
What about unassigning?

Reply:
This is not yet available in the user interface. But I'll add a commented sections that can be enabled in the future when the feature becomes available